gensrc/script/doris_builtins_functions.py

```diff
@@ -996,6 +996,7 @@
 
         [['to_days'], 'INT', ['DATEV2'], ''],
         [['time_to_sec'], 'INT', ['TIME'], ''],
+        [['sec_to_time'], 'TIME', ['INT'], ''],
```

Comment:
Also need to implement for timev2

Reply:
not support now. it will be added in another pr.